content/millennium/dstu2/medications/immunization.md

```diff
@@ -7,6 +7,32 @@ title: Immunization | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Immunization resource includes the view of current and historical administration of vaccinations to a patient in all healthcare settings. This resource contains the functionality to query a patient's immunization history.
+
+Detailed administration records may be found in MedicationAdministration, while the Immunization resource would represent the known vaccination history regardless of where the administration itself was done.
+
+An immunization reaction may indicate an allergy or intolerance. If so, a separate AllergyIntolerance resource instance should be created as well. 
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
+* [Vaccine administered](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.vaccineCode){:target="_blank"}
+* [Administration date/time](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.date){:target="_blank"}
+* [Status](http://hl7.org/fhir/immunization-definitions.html#Immunization.status){:target="_blank"}
+* [Patient](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.patient){:target="_blank"}
+* [Was not given](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.wasNotGiven){:target="_blank"}
+* [Reason not given](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.explanation.reasonNotGiven){:target="_blank"}
+* [Patient encounter](http://hl7.org/fhir/immunization-definitions.html#Immunization.encounter){:target="_blank"}
+* [Who administered](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.performer){:target="_blank"}
+* [Administration site](http://hl7.org/fhir/immunization-definitions.html#Immunization.location){:target="_blank"}
+* [Administration route](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.route){:target="_blank"}
+* [Vaccine manufacturer](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.manufacturer){:target="_blank"}
+* [Lot number](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.lotNumber){:target="_blank"}
+* [Expiration date](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.expirationDate){:target="_blank"}
+* [Dosage](http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization.doseQuantity){:target="_blank"}
```

Comment:
Just double checking if this should be "Dose" or "Dosage". I can never keep the difference straight. The underlying field is "doseQuantity"